cadquery/cq.py

```diff
@@ -2770,6 +2784,45 @@ def clean(self):
             raise AttributeError(
                 "%s object doesn't support `clean()` method!" % obj.ShapeType())
         return self.newObject(cleanObjects)
+      
+    def text(self, txt, fontsize, distance, cut=True, combine=False, clean=True,
```

Comment:
Is the cut parameter combined in the function call here because you need to keep the text as-is without converting to edges/wires? I ask because I could also see making the text into wires and doing a separate, chained cut or extrude operation with it.

Reply:
OCC function returns a compound of faces. I could convert it to a compound of wires in principle, sort it and extrude again, but this feels like a detour. I also thought that the `cut` use case is quite common.